Sources/MotorControl/Views/MotorControlAssessmentView.swift

```diff
@@ -67,19 +67,13 @@ public struct MotorControlAssessmentView : View {
             else if let nodeState = state as? MotorControlInstructionState {
                 InstructionView(nodeState: nodeState)
             }
-            else if let nodeState = state as? TremorStepViewModel {
-                MotionSensorStepView(state: nodeState)
-                    .modifier(AppBackgroundListener())
-            }
             else if let nodeState = state as? TappingStepViewModel {
                 TappingStepView(state: nodeState)
                     .modifier(AppBackgroundListener())
             }
-            else if let nodeState = state as? WalkStepViewModel {
-                MotionSensorStepView(state: nodeState)
-            }
-            else if let nodeState = state as? BalanceStepViewModel {
+            else if let nodeState = state as? MotionSensorStepViewModel {
                 MotionSensorStepView(state: nodeState)
+                    .modifier(AppBackgroundListener())
```

Comment:
Is the app background listener suppose to be there for all? or just if *not* putting the phone in your pocket?